data/sql/updates/pending_db_world/rev_1697941424465768700.sql

```diff
@@ -0,0 +1,254 @@
+-- Ashbringer is buff
+DELETE
+FROM `spell_custom_attr`
+WHERE `spell_id` = 28282;
+INSERT INTO `spell_custom_attr` (`spell_id`, `attributes`) VALUES (28282, 67108864);
+
+-- It takes time to open the door with a key in Blizzard servers
+UPDATE `gameobject_template_addon` SET `flags` = 34
```

Comment:
Please set flag bits correctly

https://www.azerothcore.org/wiki/bit-and-bytes-tutorial

https://www.azerothcore.org/flag-checker/